packages/reactivity/src/computed.ts

```diff
@@ -81,22 +81,15 @@ export function computed<T>(
   debugOptions?: DebuggerOptions,
   isSSR = false
 ) {
-  let getter: ComputedGetter<T>
-  let setter: ComputedSetter<T>
-
+  const _setter = __DEV__
+    ? () => {
+        console.warn('Write operation failed: computed value is readonly')
+      }
+    : NOOP
   const onlyGetter = isFunction(getterOrOptions)
-  if (onlyGetter) {
-    getter = getterOrOptions
-    setter = __DEV__
-      ? () => {
-          console.warn('Write operation failed: computed value is readonly')
-        }
-      : NOOP
-  } else {
-    getter = getterOrOptions.get
-    setter = getterOrOptions.set
-  }
-
+  
+  const getter = onlyGetter ? getterOrOptions : getterOrOptions.get
+  const setter = onlyGetter ? _setter : getterOrOptions.set
```

Comment:
If `onlyGetter` judgment is false, then `_setter` statement is meaningless ah.
And after changing to this writing style, `onlyGetter` needs to be judged twice.